Python 3 compatibility issues

[flake8](http://flake8.pycqa.org) testing of https://github.com/m4ll0k/Galileo on Python 3.7.0

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./lib/utils/banner.py:11:8: E999 SyntaxError: invalid syntax
	print R%1+"\t   _____       _ _ _                        "+E
       ^
./lib/utils/decoder.py:13:24: F821 undefined name 'unicode'
	if not isinstance(obj,unicode):
                       ^
./lib/utils/decoder.py:14:10: F821 undefined name 'unicode'
		return unicode(obj)         ^
1     E999 SyntaxError: invalid syntax
2     F821 undefined name 'unicode'
3
```

--- a/lib/utils/banner.py
+++ b/lib/utils/banner.py
@@ -5,16 +5,18 @@
 # @descr : Web Application Audit Framework
 # @author: Momo Outaadi
 
+from __future__ import print_function
+
 from lib.utils.colors import *
 
 def banner():
-	print R%1+"\t   _____       _ _ _                        "+E
-	print R%1+"\t  / ____|     | (_) |                       "+E
-	print R%1+"\t | |  __  __ _| |_| | ___  ___              "+E
-	print R%1+"\t | | |_ |/ _` | | | |/ _ \/ _ \             "+E
-	print R%1+"\t | |__| | (_| | | | |  __/ (_) |            "+E
-	print R%1+"\t  \_____|\__,_|_|_|_|\___|\___/             "+E
-	print Y%1+"\t     Web App Audit Framework              \n"+E
-	print "       -=[      Galileo v0.1.0       ]=-          "
-	print "+ ---====[    Momo Outaadi M4ll0k    ]====---- +  "
-	print "+ ---====[ https://github.com/m4ll0k ]====---- +\n"
+	print(R%1+"\t   _____       _ _ _                        "+E)
+	print(R%1+"\t  / ____|     | (_) |                       "+E)
+	print(R%1+"\t | |  __  __ _| |_| | ___  ___              "+E)
+	print(R%1+"\t | | |_ |/ _` | | | |/ _ \/ _ \             "+E)
+	print(R%1+"\t | |__| | (_| | | | |  __/ (_) |            "+E)
+	print(R%1+"\t  \_____|\__,_|_|_|_|\___|\___/             "+E)
+	print(Y%1+"\t     Web App Audit Framework              \n"+E)
+	print("       -=[      Galileo v0.1.0       ]=-          ")
+	print("+ ---====[    Momo Outaadi M4ll0k    ]====---- +  ")
+	print("+ ---====[ https://github.com/m4ll0k ]====---- +\n")

--- a/lib/utils/decoder.py
+++ b/lib/utils/decoder.py
@@ -5,9 +5,16 @@
 # @descr : Web Application Audit Framework
 # @author: Momo Outaadi
 
+try:
+	unicode        # Python 2
+except NameError:
+	unicode = str  # Python 2
+
+
 def to_str(obj):
 	if not isinstance(obj,str):
 		return str(obj)
+
 
 def to_unicode(obj):
 	if not isinstance(obj,unicode):

